config/database.yml

```diff
@@ -13,9 +13,10 @@ mysql: &MYSQL
   adapter: mysql2
   username: <%= ENV.fetch('DBUSERNAME','root') %>
   password: <%= ENV['DBPASSWORD'] %>
-  encoding: utf8
+  encoding: utf8mb4
+  collation: utf8mb4_unicode_ci
   properties:
-    characterSetResults: utf8
+    characterSetResults: utf8mb4
```

Comment:
This is what sets the defaults for the database, but it only happens on db:create, so I've also included a migration to update any dev databases.